lib/compiler-llvm/src/compiler.rs

```diff
@@ -72,6 +71,25 @@ impl SymbolRegistry for ShortNames {
     }
 }
 
+struct SendModule<'a> {
+    module: Arc<Mutex<Module<'a>>>,
+}
+impl<'a> SendModule<'a> {
+    fn new(module: Module<'a>) -> Self {
+        SendModule {
+            module: Arc::new(Mutex::new(module)),
+        }
+    }
+    fn into_inner(self) -> Module<'a> {
+        Arc::try_unwrap(self.module).unwrap().into_inner().unwrap()
+    }
+    fn link_in_module(&self, linked: SendModule<'a>) {
+        self.module.lock().unwrap().link_in_module(linked.into_inner());
+    }
+}
+
+unsafe impl Send for SendModule<'_> {}
+
```

Comment:
I created this struct in order to be able to use LLVM modules across threads easily.

Reply:
This is a good idea, nearly every (are there _any_ exceptions?) LLVM type is Send, we should add them to inkwell.